service/Microsoft.Azure.Devices/QueryResponse.cs

```diff
@@ -14,7 +14,7 @@ public class QueryResponse<T> : IEnumerable<T>
     {
         readonly IEnumerable<T> queryResults;
 
-        internal QueryResponse(IEnumerable<T> queryResults, string continuationToken)
+        public QueryResponse(IEnumerable<T> queryResults, string continuationToken)
```

Comment:
because this class is referenced by both iothub and dps service clients.  In order for dps service clients to referenced it in the iothub project, we have to expose it...

---
In reply to: [141730370](https://github.com/Azure/azure-iot-sdk-csharp/pull/174#discussion_r141730370) [](ancestors = 141730370)